src/extension.ts

```diff
@@ -180,6 +180,9 @@ async function init(
 		"startup" : {}
 	*/
 	telemetry.sendTelemetryEvent('startup');
+
+	const experimentationService = await createExperimentationService(context, telemetry);
+	experimentationService.isCachedFlightEnabled('githubaa');
```

Comment:
Is that string correct? (Just asking because it seems an odd one)

Reply:
yeah, good question. this is based on the scorecard we created so is correct, but I think for an actual experiment we would create a more meaningful name